src/core_plugins/kibana/public/management/sections/settings/components/field/field.js

```diff
@@ -475,7 +476,22 @@ export class Field extends PureComponent {
       <Fragment>
         <EuiSpacer size="s" />
         <EuiText size="xs">
-          Default: <EuiCode>{this.getDisplayedDefaultValue(type, defVal)}</EuiCode>
+          { type === 'json' ? (
+            <Fragment>
+              Default:
+              <EuiCodeBlock
+                language="js"
```

Comment:
should this be json instead of js?

Reply:
good call, for some reason I thought both would highlight the same, but `json` is a little cleaner. updated.